packages/fluentui/accessibility/src/behaviors/Button/buttonBehavior.ts

```diff
@@ -7,8 +7,7 @@ export const buttonBehavior: Accessibility<ButtonBehaviorProps> = props => {
       root: {
         role: props.as === 'button' ? undefined : 'button',
         tabIndex: props.as === 'button' || props.disabled ? undefined : 0,
-        disabled: props.disabled && !props.loading ? (props.as === 'button' ? true : undefined) : undefined,
```

Comment:
Now the `disabled` prop no longer works

Reply:
great catch :) thank you very much for proper/deep look into it :)
I revert adding disabled attribute via prop.